decidim-proposals/lib/decidim/proposals.rb

```diff
@@ -25,6 +25,18 @@ module Proposals
     config_accessor :similarity_limit do
       10
     end
+
+    # Public Setting that defines how many proposals will be shown in the
+    # participatory_space_highlighted_elements view hook
+    config_accessor :participatory_space_highlighted_proposals_limit do
+      4
+    end
+
+    # Public Setting that defines how many proposals will be shown in the
+    # process_group_highlighted_elements view hook
+    config_accessor :process_group_highlighted_proposals_limit do
+      3
```

Comment:
These two new settings have different values, is this intended?

Reply:
Yes! It is intended.